audio/static/html/audio_edit.html

```diff
@@ -2,8 +2,20 @@
   <ul class="list-input settings-list">
     <li class="field comp-setting-entry is-set">
       <div class="wrapper-comp-setting">
-        <label class="label setting-label" for="audio_src">Audio Source Location</label>
-        <input class="input setting-input" name="audio_src" id="audio_src" value="{src}" type="text" />
+        <label class="label setting-label" for="audio_src">Audio Source (.ogg)</label>
+        <input placeholder="http://audio.com/my-audio.ogg" class="input setting-input" name="audio_src" id="audio_src" value="{src}" type="text"/>
+      </div>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="audio_src">Audio Source (.mp3)</label>
```

Comment:
`for` attribute should be `audio_src_downloadable `

Reply:
ok